Gopkg.toml

```diff
@@ -0,0 +1,17 @@
+
+[[constraint]]
+  name = "github.com/bitrise-io/go-utils"
+  branch = "master"
+
+[[constraint]]
+  branch = "feature/app-icon"
+  name = "github.com/bitrise-io/bitrise-init"
+
+# Need the previous released version, as bigger bitrise format version is not supported by website
+[[override]]
+  name = "github.com/bitrise-io/bitrise"
+  version = "1.28.0"
```

Comment:
in case of our own packages pls lock for master, other than that is this package used?

Reply:
Check the comment